priv/xcodegen/Sources/TemplateApp/ContentView.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ContentView.swift
+//  TemplateApp
+//
+
+import SwiftUI
+import LiveViewNative
+
+struct ContentView: View {
+    var body: some View {
+        LiveView(
+            .automatic(development: .localhost, production: .custom(URL(string: "example.com")!)),
+            configuration: .init(connectParams: { _ in ["apns_token": "tokenValue"] })
```

Comment:
Not sure if `configuration` should be included by default, this line doesn't send any meaningful data.